react-frontend/src/components/WorkoutCard.js

```diff
@@ -34,35 +19,32 @@ function WorkoutCard(props) {
                          >
                               {date}
                          </Card.Title>
-                         <Card.Text>
-                              <Table>
-                                   <thead>
+                         <Table>
+                              <thead>
+                                   <tr>
+                                        <th style={{ width: "40%" }}>
+                                             Exercise
+                                        </th>
+                                        <th style={{ width: "10%" }}>Weight</th>
+                                        <th style={{ width: "10%" }}>Sets</th>
+                                        <th style={{ width: "10%" }}>Reps</th>
+                                   </tr>
+                              </thead>
+                              <tbody>
+                                   {props.item && (
                                         <tr>
-                                             <th style={{ width: "40%" }}>
-                                                  Workout
-                                             </th>
-                                             <th style={{ width: "10%" }}>
-                                                  Weight
-                                             </th>
-                                             <th style={{ width: "10%" }}>
-                                                  Sets
-                                             </th>
-                                             <th style={{ width: "10%" }}>
-                                                  Reps
-                                             </th>
+                                             <td>{props.item.lift_name}</td>
```

Comment:
Won't display multiple weightlifts for a single day. May have to implementing sorting by dates and mapping (workoutData in Home.js )